installer/Migration/V5_1_0/LangStringHelper.php

```diff
@@ -0,0 +1,253 @@
+<?php
```

Comment:
@SharukaNP won't this be conflicting with your [PR](https://github.com/Super-Chama/orangehrm/pull/52)? Please advise how to resolve this ?

Reply:
Ah no I can rebase once his PR is merged vise-versa depending on whose PR is merged first